packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.classNames.ts

```diff
@@ -0,0 +1,22 @@
+import { memoizeFunction } from '../../Utilities';
+import { mergeStyleSets } from '../../Styling';
+
+export interface IVerticalDividerClassNames {
+  wrapper: string;
+  divider: string;
+}
+
+export const getDividerClassNames = memoizeFunction((): IVerticalDividerClassNames => {
+  return mergeStyleSets({
+    wrapper: {
+      display: 'inline-flex',
+      height: '100%',
+      alignItems: 'center'
+    },
+    divider: {
+      width: 1,
+      height: '100%',
+      backgroundColor: '#C8C8C8'
```

Comment:
This hardcoded color should be a palette color. Otherwise it can't be themed.

Reply:
Oops missed that. I had meant to come back to that.